### PR TITLE
CNV - updating registry procedures for VMware import

### PR DESCRIPTION
**Pull request by apinnick**
CP for 4.2, 4.3, and 4.4.

### CODE REVIEW

File: modules/cnv-creating-vddk-image.adoc [hunk 1]
```diff
@@ -47,18 +43,20 @@ EOF
...
+. Push the image to the registry:
 +
 ----
-$ docker push <server/username/reponame:tag> <1>
+$ podman push <registry_route_or_server_path>/vddk:<tag>
 ----
-<1> Specify your repository details.
 
-. Ensure that the image is accessible to your {CNVProductName} environment.
+. Ensure that the image is accessible to your {VirtProductName} environment.
```
Comment (kalexand-rh) [inline modules/cnv-creating-vddk-image.adoc:59]: Is this attribute right in 4.4?

**Review by kalexand-rh — COMMENTED** [review]
Two small requests, then it LGTM.

File: cnv/cnv_virtual_machines/cnv_importing_vms/cnv-importing-vmware-vm.adoc [hunk 2]
```diff
@@ -12,18 +12,46 @@ If you import a VMware template, the wizard creates a virtual machine based on t
...
 :!FeatureName:
 
-include::modules/cnv-creating-vddk-image.adoc[leveloffset=+1]
+The import process uses the VMware Virtual Disk Development Kit (VDDK) to copy the VMware virtual disk. You can download the VDDK SDK, build a VDDK image, upload image to your image registry, and add it to the `v2v-vmware` ConfigMap.
+
+You can import the VMware VM with the virtual machine wizard and then update the virtual machine's network name.
+
+== Configuring an image registry for the VDDK image
+
+You can configure either an internal {product-title} image registry or a secure external image registry for the VDDK image.
```
[the remark this answers is not in the document]
Comment (apinnick) [reply cnv/cnv_virtual_machines/cnv_importing_vms/cnv-importing-vmware-vm.adoc:21]: It is nearly impossible to anticipate why a user would want to use one over the other, especially when we're talking about migration. There are too many factors involved for us to make any kind of recommendation. It depends solely on the user's environment.

File: modules/cnv-creating-vddk-image.adoc (hunk 1 above)
Comment (apinnick) [reply modules/cnv-creating-vddk-image.adoc:59]: No, it's not. Will fix.

**Comment by apinnick** [comment]
@kalexand-rh Sorry, my bad! This should not be CPed to 4.2. Thanks!

**Pull request merged**